main.go

```diff
@@ -189,6 +189,10 @@ type NavElem struct {
 	CanPOST bool
 }
 
+type LoginResponse struct {
```

Comment:
you can move this type where it belongs (login.go) doesnt need to be in the same file as where it's used